api/database/models.py

```diff
@@ -19,4 +23,6 @@ class Brevet(Document):
 		start_time: MongoEngine datetime field, required
 		checkpoints: MongoEngine list field of Checkpoints, required
     """
-    pass
+    distance = FloatField(required=True)
+    start_time = StringField(required=True)
+    checkpoint = EmbeddedDocumentListField(Checkpoint)
```

Comment:
API models looks good, pretty much the same thing I did. Only difference I would make would be to set checkpoint to required
```suggestion 
    checkpoint = EmbeddedDocumentListField(Checkpoint, required=True)
 ```